### PR TITLE
Fix theme path in JupyterLab builder

### PR DESCRIPTION
**Pull request by martinRenou**
Signed-off-by: martinRenou <martin.renou@gmail.com>

## Code changes

This fixes the path to the theme CSS in the config

## Backwards-incompatible changes

cc. @jtpio

### CODE REVIEW

**Comment by jtpio** [comment]
Nice, thanks @martinRenou!

Something noticed when porting the theme cookiecutter to lab 3: https://github.com/jupyterlab/theme-cookiecutter/pull/19

File: builder/src/build.ts [hunk 1]
```diff
@@ -199,7 +199,7 @@ export namespace Build {
       themeConfig.push({
         mode: 'production',
         entry: {
-          index: path.join(name, themePath)
+          index: path.join(packagePath, themePath)
```
Comment (jasongrout) [inline builder/src/build.ts:202]: For consistency, can we use `packageDir`, which is used above? I know they should be the same, but still, for consistency with usage above.

Comment (martinRenou) [reply builder/src/build.ts:202]: Should we remove one of the variables to reduce confusion? If they are always the same.

**Review by jtpio — APPROVED** [review]
Thanks!

**Pull request merged**